src/Client.js

```diff
@@ -1028,8 +1028,8 @@ class Client extends EventEmitter {
             if(!window.Store.Conn.canSetMyPushname()) return false;
 
             if(window.Store.MDBackend) {
-                // TODO
-                return false;
+                await window.Store.Settings.setPushname(displayName);
```

Comment:
seems like that function `window.Store.Settings.setPushname` changes the pushname only locally
i tested the bellow method and works great

```suggestion
                await window.mR.modules[858640].default(displayName);
```